lib/fpm/package/deb.rb

```diff
@@ -18,6 +18,7 @@ class FPM::Package::Deb < FPM::Package
     :after_install      => "postinst",
     :before_remove      => "prerm",
     :after_remove       => "postrm",
+    :after_remove_purge => "postrm",
```

Comment:
`apt` calls this 'purge' not "remove purge" -- thoughts on just having this called `after_purge` ?